src/Controllers/LoggerController.js

```diff
@@ -16,7 +16,7 @@ export const LogOrder = {
   ASCENDING: 'asc',
 };
 
-const logLevels = ['error', 'warn', 'info', 'debug', 'verbose', 'silly'];
+export const logLevels = ['error', 'warn', 'info', 'debug', 'verbose', 'silly'];
```

Comment:
So we don't offer a `none` log level in this PR to disable logging for a specific log entry type, correct?

Reply:
Correct, I think there is enough levels to handle logs properly.
For example, in my company, we would use the 'debug' level for triggers because we feel this is not relevant for production. And this is really the point of this PR : prevent the log file to grow dramatically with useless informations that could drown out those that actually are.
Moreover, if this is not enough, it remains the 2 lower levels 'verbose' and 'silly', wich are, I guess, perfectly for that.
So, adding one more option seems to be a bit overkill and I would suggest to stay simple.

Reply:
Sounds reasonable, good choice I think